--- Glossary.orig	2006-04-09 12:35:52.000000000 +0200
+++ Glossary	2006-04-09 12:36:00.000000000 +0200
@@ -3537,7 +3537,7 @@ make_set_make (make.U):
 	make_set_make="MAKE=$make"	# if it doesn't.
 
 	This uses a comment character to distinguish a
-	'set' value (from a previous config.sh or Configure -D option)
+	'set' value (from a previous config.sh or Configure '-D' option)
 	from an uncomputed value.
 
 mallocobj (mallocsrc.U):
@@ -3654,7 +3654,7 @@ myuname (Oldconfig.U):
 	whole thing is then lower-cased.
 
 n (n.U):
-	This variable contains the -n flag if that is what causes the echo
+	This variable contains the '-n' flag if that is what causes the echo
 	command to suppress newline.  Otherwise it is null.  Correct usage is
 	$echo $n "prompt for a question: $c".
 
@@ -4097,9 +4097,9 @@ sh (sh.U):
 	/bin/sh, though it's possible that some systems will have /bin/ksh,
 	/bin/pdksh, /bin/ash, /bin/bash, or even something such as
 	D:/bin/sh.exe.
-	This unit comes before Options.U, so you can't set sh with a -D
+	This unit comes before Options.U, so you can't set sh with a '-D'
 	option, though you can override this (and startsh)
-	with -O -Dsh=/bin/whatever -Dstartsh=whatever
+	with '-O -Dsh=/bin/whatever -Dstartsh=whatever'
 
 shar (Loc.U):
 	This variable is defined but not used by Configure.
@@ -4350,7 +4350,7 @@ spackage (package.U):
 
 spitshell (spitshell.U):
 	This variable contains the command necessary to spit out a runnable
-	shell on this system.  It is either cat or a grep -v for # comments.
+	shell on this system.  It is either cat or a grep '-v' for # comments.
 
 sPRId64 (quadfio.U):
 	This variable, if defined, contains the string used by stdio to
